gramofonsite/urls.py

```diff
@@ -7,8 +7,16 @@
 
 app_name = 'gramofonsite'
 urlpatterns = [
-    path('', views.gramofonsite_list, name='gramofonsite_list'), 
-    path('gramofon-morska', views.morska, name='gramofon-morska'),
+    path('gramofon-lokal', views.gramofonsite_list, name='gramofonsite_list'), 
```

Comment:
tutaj powinno być chyba ''  bo na głownej stronie jak się wchodzi to się nic nie wyświetli